src/renderer/entities/multisig/model/multisigs-model.ts

```diff
@@ -32,27 +31,34 @@ import { multisigUtils } from '../lib/mulitisigs-utils';
 const MULTISIG_DISCOVERY_TIMEOUT = 30000;
 
 const subscribe = createEvent();
+const stop = createEvent();
 const request = createEvent<AnyAccount[]>();
 
+const createWallets = attach({ effect: walletModel.createWallets });
+
 const $multisigAccounts = walletModel.$allWallets
   .map(walletUtils.getAllAccounts)
   .map((accounts) => accounts.filter(accountUtils.isMultisigAccount));
 
-const { tick: pollingRequest } = interval({
+const { tick: pollingRequest, isRunning: $isPollingRunning } = interval({
   start: subscribe,
+  stop: stop,
   timeout: MULTISIG_DISCOVERY_TIMEOUT,
 });
 
+const readyToRequest = combineEvents({
+  events: [walletModel.populate.done, accounts.populate.done, networkModel.events.connectionsPopulated],
+});
+
 const updateRequested = sample({
-  clock: [pollingRequest, networkModel.events.connectionsPopulated],
-  source: walletModel.$allWallets,
-  fn: (wallets) => {
-    const filteredWallets =
-      walletUtils.getWalletsFilteredAccounts(wallets, {
-        walletFn: (w) => !walletUtils.isWatchOnly(w) && !walletUtils.isProxied(w) && !walletUtils.isMultisig(w),
-      }) ?? [];
-
-    return walletUtils.getAllAccounts(filteredWallets);
+  clock: [pollingRequest, readyToRequest],
+  source: walletModel.$availableAccounts,
+  fn(accounts) {
```

Comment:
Any reasons for such style? In some places you use arrow function

Reply:
I'm experimenting with code style right now and this is one of the latest iteration. I think it's better to add eslint warning.